templates/community/subscribe.macros.j2

```diff
@@ -90,7 +90,7 @@
       <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
       <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_80219aa68d7bad77b9fd2eb93_7c27f7fb9a" tabindex="-1" value=""></div>
 
-      <div class="clear odd-submit-btn">
```

Comment:
I assume removing the class clear won't hurt anything (I know we don't need it), but mailchimp had it in there and it was under the line that said not to touch anything. That said, I added classes so I am just as guilty.

Reply:
I'm pretty sure it just adds a clearfix, so i'm not worried about it. We're also not using any of their direct interactions on the page, so only need to worry about the values we post - otherwise I think the markup is in our control.